src/CompatHelper.jl

```diff
@@ -1,14 +1,14 @@
 module CompatHelper
 
-import Base64
-import Dates
-import GitHub
-import HTTP
-import Pkg
-import Printf
-import TOML
-import TimeZones
-import UUIDs
+using Base64
+using Dates
+using GitHub
+using Pkg
+using Printf
+using TOML
+using TimeZones
+using UUIDs
+
```

Comment:
I guess it's fine with these packages and the function names used in this code but in general it's not NFC to change `import` to `using`.